slips_files/common/parsers/config_parser.py

```diff
@@ -27,14 +30,14 @@ def __init__(self):
 
     def read_config_file(self):
         """
-        reads slips configuration file, slips.conf is the default file
+        reads slips configuration file, slips.conf/slips.yaml is the default file
         """
         config = configparser.ConfigParser(
             interpolation=None, comment_prefixes="#"
         )
         try:
             with open(self.configfile) as source:
-                config.read_file(source)
+                yaml.safe_load(source)
```

Comment:
now after parsing the file, the return of this line is never used, therefore the config.yaml is never used by slips